jsk_robot_common/jsk_robot_startup/README.md

```diff
@@ -5,6 +5,62 @@ jsk_robot_startup
 
 see [lifelog/README.md](lifelog/README.md)
 
+## t265_odometry_transformer.py
+
+This node publishes odometry topics and TF with raw odometry topics of Realsense T265. Realsense T265 can publish odometry TF in camera frames. This node helps to use them in robot base_link frames.
+
+![t265_odometry_transformer](https://user-images.githubusercontent.com/9410362/172747852-9d78d710-528d-4c53-ba32-9af8021ce7eb.png)
+
+### Subscribers
+
+- `~odom_in` (type: nav_msgs/Odometry)
+
+Raw odometry topic from Realsense T265
+
+### Publishers
+
+- `~odom_out` (type: nav_msgs/Odometry)
+
+Transformed odometry topic.
+
+- `/tf` (type: tf2_msgs/TFMessage)
+
+### Parameters
+
+- `~frame_id_base_link` (type: `string`, default: `base_link`)
+
+Frame ID of robot base_link. It is assumed that this link is top frame of TF tree because TF from odom to this link will be broadcasted.
+
+- `~frame_id_odom` (type: `string`, default: `odom`)
```

Comment:
```suggestion
- `~odom_frame_id` (type: `string`, default: `odom`)
```

IMO, this word order is better, but what do you think?

Reply:
https://github.com/introlab/rtabmap_ros/blob/1e28de0b38d20410d47fb9fb5d4c8d96f1590c5d/launch/rgbd_mapping.launch#L40